src/Pages/minji/Login/Login.js

```diff
@@ -1,13 +1,71 @@
-import React, { Component } from 'react';
+import React from "react";
+import "./Login.scss";
 
-class Login extends Component {
-    render() {
-        return (
-            <div>
-                
-            </div>
-        );
+class LoginMinji extends React.Component {
+  constructor() {
+    super();
+    this.state = { id: "", password: "", color: "rgb(185, 216, 256)" };
+  }
```

Comment:
```suggestion
  constructor() {
    super();
    this.state = { 
      id: "", 
      password: "", 
      color: "rgb(185, 216, 256)" 
    };
  }
```
- 가독성을 위해 줄바꿈 처리 해주시기 바랍니다. (ESLint, Prettier 설치 해주세요~)
- color 값을 직접 다루기 보다는 isBtnActive: true/false 값으로 버튼의 활성화 상태를 state에서 관리해주시는 것이 좋습니다!

Reply:
prettier설치 되었는데 정렬을 해도 저 상태에서 정리가 안되네요ㅠㅠ 정리하겠습니다 감사합니다!